sphinx/builders/linkcheck.py

```diff
@@ -134,6 +136,10 @@ def check_uri():
             # split off anchor
             if '#' in uri:
                 req_url, anchor = uri.split('#', 1)
+                for rex in self.anchors_ignore:
+                    if rex.match(uri):
```

Comment:
I think it depends on the use case.  Could you maybe elaborate on your motivation for this?

I can guess two motivations:
* Skipping domains ("I know example.com uses weird anchors, so let's ignore them there.")
* Skipping a kind of anchor ("I know ExampleJS uses anchors of this weird style, so let's ignore them.")

I'm concerned about this munging both into one, because it possibly makes forging a regexp for either of the use cases harder:
* `/example.com/` would also match `http://sphinx-doc.com/#example.com`
* `/#page=/` would also match `http://example.com/#static#page=foo` (maybe a bit of a strawman.)